packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/instance/attributeStartingAt..st

```diff
@@ -0,0 +1,10 @@
+attributes
+attributeStartingAt: aMarkdownDelimiterEmphasis
```

Comment:
I found this method hard to understand, as it puts the receiver into the role of a closer and the arguments as opener. Maybe it would help to reverse this to `attributeTo: closingMarkdownDelimiterEmphasis`. Nevertheless, a semantic-typed arg name would help too

Reply:
We decided not to reverse it but to change the parameter name to `openingMarkdownDelimiterEmphasis`